cpp/ql/lib/change-notes/2024-01-30-preproc-block.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
```

Comment:
I think this should actually be categorized as `feature`: https://github.com/github/codeql/blob/main/docs/change-notes.md#library-pack-change-categories

Reply:
Good point.  Updated.